test/models/blogs.tape.js

```diff
@@ -1,25 +1,32 @@
 'use strict'
 
-var test = require('tape')
+var test = require('tape-catch')
 var Blog = require('../../models/Blogs')
+var uuid = require('uuid')
 
-test('#save()', function (t) {
+var db = require('../db-connect')()
+
+test('Blogs #save()', function (t) {
   t.plan(2)
 
+  var uniquePostName = uuid.v4()
+
   let blogpost = new Blog({
-    title: 'Jonny Foobar',
+    title: uniquePostName,
```

Comment:
Yeah, I was doing that in my `mocha` tests but not in `tape` because I got lazy and sleepy while on the train to Shenzhen.